lib/stages/pulsarPostProcess.cpp

```diff
@@ -249,7 +258,7 @@ void pulsarPostProcess::main_thread() {
 
                 // Fill the headers of the new buffer
                 fill_headers((unsigned char*)out_frame, &vdif_header, fpga_seq_num, &time_now,
-                             psr_coord, (uint16_t*)freq_ids);
+                             psr_coord, (uint16_t*)thread_ids);
```

Comment:
Why is this case necessary? it's a `uint16_t[]`, which I thought can be passed as a pointer?

Reply:
I agree, not necessary.  I removed those casts.